syntax = "proto3";

message Record {
  uint64 frame_num = 1;
  NpArray net_data = 2;
  NpArray lidar_data = 3;
}

message NpArray {
  string dtype = 1;
  repeated uint32 shape = 3;
  bytes data = 4;
}
